head.html

```diff
@@ -1,6 +1,8 @@
-<meta name="viewport" content="width=device-width, initial-scale=1"/>
-<link rel="preload" href="/query-index.json" as="fetch" type="application/json" crossorigin="anonymous">
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<link rel="preload" href="/query-index.json" as="fetch" type="application/json" crossorigin="anonymous" />
+<link rel="preload" href="/nav.plain.html" as="fetch" type="application/json" crossorigin="anonymous" />
```

Comment:
Are you sure this works ? It's HTML not JSON ...

Reply:
yes it still worked. Also works without a type at all.